core-web/libs/block-editor/src/lib/extensions/bubble-link-form/components/suggestion-page/suggestion-page.component.html

```diff
@@ -0,0 +1,25 @@
+<div *ngIf="items.length; else loading ? loadingBlock : emptyBlock">
+    <dot-suggestion-list #list>
+        <dot-suggestions-list-item
+            *ngFor="let item of items; let i = index"
+            [command]="item.command"
+            [index]="i"
+            [label]="item.label"
+            [url]="item.icon"
+            [data]="item.data"
+            [page]="true"
+        >
+        </dot-suggestions-list-item>
+    </dot-suggestion-list>
+</div>
+
+<ng-template #loadingBlock>
+    <dot-suggestion-loading-list *ngIf="loading"></dot-suggestion-loading-list>
```

Comment:
Do we need the `*ngIf="loading"` here?

Reply:
No, we don't. I forgot to remove it. :sweat_smile: